lib/process/processLauncher.js

```diff
@@ -14,6 +14,7 @@ class ProcessLauncher {
    * @return {ProcessLauncher}    The ProcessLauncher instance
    */
   constructor(options) {
+    this.env = options.env;
```

Comment:
Why do you need env?

Reply:
I'm kicking down the env passed on the embark cli to `webpackProcess.js`.

Reply:
Upon further review, this is not needed, so I removed it in a commit I just made.